### PR TITLE
Serenity update July-August 2021

### PR DESCRIPTION
**Pull request by felix-couprie**
Serenity updates for July and August 2021

Screenshot 2021-08-25 at 16 23 21

screencapture-0-0-0-0-8000-pim-serenity-updates-2021-08-html-2021-08-25-16_43_11

### CODE REVIEW

**Review by meganelepalud-akeneo — APPROVED** [review]

File: content/updates/2021-08/2-Tailored-Export.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Tailored export - **customize your product exports**
+::: meta-data type="New" features="Productivity" available="mid-August" in="EE,GE" link-to-doc="../articles/tailored-export.html"
+
+This feature consists in a new export profile, available only for our Enterprise Edition or Growth Edition users (*available soon*). It helps you to export your product information in flat files, with your own structure.
```
Comment (LaureBrosseau) [inline content/updates/2021-08/2-Tailored-Export.md:10]:
```suggestion
We are pleased to introduce a new export feature called **Tailored Exports**. Now available among the list of export jobs, Tailored Exports allow you to customize the file structure and the product content exported. Thanks to Tailored Exports, you can now easily create and export XLSX files per your needs, per your product partners (distributor, retailer...), or per channel requirements. Its straightforward interface enables you to easily select, rename and reorder columns, and even adapt the content of your attributes thanks to predefined actions like a concatenate one. This new capability will be enriched with more functionalities in the coming weeks, stay tuned!
```

File: content/updates/2021-08/2-Tailored-Export.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Tailored export - **customize your product exports**
+::: meta-data type="New" features="Productivity" available="mid-August" in="EE,GE" link-to-doc="../articles/tailored-export.html"
+
+This feature consists in a new export profile, available only for our Enterprise Edition or Growth Edition users (*available soon*). It helps you to export your product information in flat files, with your own structure.  
+For instance, let's assume your retailers or distributors need the same product information from you, but they want different structures:
```
Comment (LaureBrosseau) [inline content/updates/2021-08/2-Tailored-Export.md:11]:
```suggestion
```

File: content/updates/2021-08/2-Tailored-Export.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+* Retailer A wants an export with columns named: "Identifier", "Product Name", "Description", and "Weight (Ounces)".
```
Comment (LaureBrosseau) [inline content/updates/2021-08/2-Tailored-Export.md:12]:
```suggestion
```

File: content/updates/2021-08/2-Tailored-Export.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+* Retailer B wants an export with columns named: "Name in French", "Name in English", "Description 1", "ID", and "Weight (Kilograms)".
```
Comment (LaureBrosseau) [inline content/updates/2021-08/2-Tailored-Export.md:13]:
```suggestion
```

File: content/updates/2021-08/2-Tailored-Export.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
```
Comment (LaureBrosseau) [inline content/updates/2021-08/2-Tailored-Export.md:14]:
```suggestion
```

File: content/updates/2021-08/2-Tailored-Export.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+Previously, you had to export your product information thanks to a regular product export profile, and manually modify your spreadsheet to reorder the columns, change their name, convert the measurements, for as many retailers and distributors that needed it.
```
Comment (LaureBrosseau) [inline content/updates/2021-08/2-Tailored-Export.md:15]:
```suggestion
```

File: content/updates/2021-08/2-Tailored-Export.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+This feature consists in a new export profile, available only for our Enterprise Edition or Growth Edition users (*available soon*). It helps you to export your product information in flat files, with your own structure.  
+For instance, let's assume your retailers or distributors need the same product information from you, but they want different structures:
+* Retailer A wants an export with columns named: "Identifier", "Product Name", "Description", and "Weight (Ounces)".
+* Retailer B wants an export with columns named: "Name in French", "Name in English", "Description 1", "ID", and "Weight (Kilograms)".
+
+Previously, you had to export your product information thanks to a regular product export profile, and manually modify your spreadsheet to reorder the columns, change their name, convert the measurements, for as many retailers and distributors that needed it.  
+Now, prepare the work from Akeneo PIM creating a Tailored Export profile per your retailer's or distributor's requirements and use it as often as necessary. **You no longer need to manually edit your flat files as soon as you need to share product information.**
```
Comment (LaureBrosseau) [inline content/updates/2021-08/2-Tailored-Export.md:16]:
```suggestion
```

File: content/updates/2021-08/2-Tailored-Export.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
```
Comment (LaureBrosseau) [inline content/updates/2021-08/2-Tailored-Export.md:17]:
```suggestion
```

File: content/updates/2021-08/2-Tailored-Export.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+![Tailored Export - Structure](../img/TailoredExport_Structure.png)
+
+In this release, you'll be able to:
```
Comment (LaureBrosseau) [inline content/updates/2021-08/2-Tailored-Export.md:20]:
```suggestion
```

File: content/updates/2021-08/2-Tailored-Export.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+* Name your columns & to reorder them
```
Comment (LaureBrosseau) [inline content/updates/2021-08/2-Tailored-Export.md:21]:
```suggestion
```

File: content/updates/2021-08/2-Tailored-Export.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+* Add system properties (groups, associations, parent, family, category etc.)
```
Comment (LaureBrosseau) [inline content/updates/2021-08/2-Tailored-Export.md:22]:
```suggestion
```

File: content/updates/2021-08/2-Tailored-Export.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+* Select any category (not only linked to a channel)
```
Comment (LaureBrosseau) [inline content/updates/2021-08/2-Tailored-Export.md:23]:
```suggestion
```

File: content/updates/2021-08/2-Tailored-Export.md [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+* Concatenate (up to 4 sources in one column)
```
Comment (LaureBrosseau) [inline content/updates/2021-08/2-Tailored-Export.md:24]:
```suggestion
```

File: content/updates/2021-08/2-Tailored-Export.md [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+* Paste headers from a spreadsheet to create your columns
```
Comment (LaureBrosseau) [inline content/updates/2021-08/2-Tailored-Export.md:25]:
```suggestion
```

File: content/updates/2021-08/2-Tailored-Export.md [hunk 15]
```diff
@@ -0,0 +1,32 @@
...
+* Search for a column
+* Replace the values for the Yes/No attribute
+* Use default value when empty
```
Comment (LaureBrosseau) [inline content/updates/2021-08/2-Tailored-Export.md:28]:
```suggestion
```

File: content/updates/2021-08/2-Tailored-Export.md [hunk 16]
```diff
@@ -0,0 +1,32 @@
...
+![Tailored Export - Structure](../img/TailoredExport_Structure.png)
+
+In this release, you'll be able to:
+* Name your columns & to reorder them
+* Add system properties (groups, associations, parent, family, category etc.)
+* Select any category (not only linked to a channel)
+* Concatenate (up to 4 sources in one column)
+* Paste headers from a spreadsheet to create your columns
+* Search for a column
+* Replace the values for the Yes/No attribute
```
Comment (LaureBrosseau) [inline content/updates/2021-08/2-Tailored-Export.md:27]:
```suggestion
```

File: content/updates/2021-08/2-Tailored-Export.md [hunk 17]
```diff
@@ -0,0 +1,32 @@
...
+
+![Tailored Export - Structure](../img/TailoredExport_Structure.png)
+
+In this release, you'll be able to:
+* Name your columns & to reorder them
+* Add system properties (groups, associations, parent, family, category etc.)
+* Select any category (not only linked to a channel)
+* Concatenate (up to 4 sources in one column)
+* Paste headers from a spreadsheet to create your columns
+* Search for a column
```
Comment (LaureBrosseau) [inline content/updates/2021-08/2-Tailored-Export.md:26]:
```suggestion
```

File: content/updates/2021-08/2-Tailored-Export.md [hunk 18]
```diff
@@ -0,0 +1,32 @@
+---
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Tailored export - **customize your product exports**
+::: meta-data type="New" features="Productivity" available="mid-August" in="EE,GE" link-to-doc="../articles/tailored-export.html"
```
[the remark this answers is not in the document]
Comment (felix-couprie) [reply content/updates/2021-08/2-Tailored-Export.md]: Hello Laure, you're right, the feature is only available in Serenity now, and should be available later in Growth, but not before end of the month.

File: content/updates/index.json [hunk 19]
```diff
@@ -1,4 +1,9 @@
 {
+  "2021-08": {
+    "title": "July & August 2021 Serenity updates",
+    "description": "Lorem ipsum",
```
[the remark this answers is not in the document]
Comment (meganelepalud-akeneo) [reply content/updates/index.json:4]: Will be updated by Laure or Camille :)

**Pull request merged**